anchore_manager/util/config.py

```diff
@@ -0,0 +1,63 @@
+import copy
+import os
+from . import logging
+
+DEFAULT_CONFIG = {
+    'jsonmode': False,
+    'debug': False,
+    'configdir': '/config'
+}
+
+
+def setup_config(cli_opts):
+    """
+    Initialize the config from the default
+    :param cli_opts:
+    :return:
+    """
+    ret = copy.copy(DEFAULT_CONFIG)
+
+    settings = {}
+
+    # load environment if present
+    try:
+        for e in ['ANCHORE_CLI_JSON', 'ANCHORE_CLI_DEBUG', 'ANCHORE_CONFIG_DIR']:
+            if e in os.environ:
+                settings[e] = os.environ[e]
+    except Exception as err:
+        raise err
+
+    # load cmdline options
+    try:
+        if cli_opts['json']:
+            settings['ANCHORE_CLI_JSON'] = "y"
+
+        if cli_opts['debug']:
+            settings['ANCHORE_CLI_DEBUG'] = "y"
+
+        if cli_opts['configdir']:
+            settings['ANCHORE_CONFIG_DIR'] = cli_opts['configdir']
+
+    except Exception as err:
+        raise err
+
+    try:
+        if 'ANCHORE_CLI_JSON' in settings:
+            if settings['ANCHORE_CLI_JSON'].lower() == 'y':
```

Comment:
this can be simplified with a get:
```suggestion  
        if settings.get('ANCHORE_CLI_JSON', '').lower() == 'y':
```